experimenter/experimenter/nimbus_ui_new/templates/common/header.html

```diff
@@ -38,6 +38,12 @@
           Help
         </a>
       </li>
+      <li class="nav-item">
+        <a href="/legacy/" class="nav-link" target="_blank" rel="noreferrer">
+          <i class="fa-solid fa-ghost"></i>
```

Comment:
I know fontawesome suggests using `<i>` for "icon", but its bad semantic HTML. We should use `<span>` (but we can do that in another change if we want to).